data/npc/lib/npc.lua

```diff
@@ -132,3 +132,25 @@ end
 function Player.getTotalMoney(self)
 	return self:getMoney() + self:getBankBalance()
 end
+
+function isValidMoney(money)
+	return isNumber(money) and money > 0 and money < 4294967296
```

Comment:
Should probably explain where the max number comes from

Reply:
idk either, maybe @ninjalulz or @TheSumm or @WibbenZ can be able to explain that since they worked on ORTS

Reply:
ok I went trough  otland search and found [this](https://otland.net/threads/question-money-bug-hardcoders-required-repping-6-bars.58738/), I don't know how to test if this is still an issue and/or should we remove the max number, but IMO it does no harm, should we just add a "-- max value of uint32_t" comment?

Reply:
In our `schema.sql`, the column type is bigint(20). You can check the source in C++ what's the variable type there but I guess it's no longer uint32. Obviously you could just test as well. I don't think this limit is applicable anymore.

Reply:
yes, now we use a int64_t variable type and I was able to deposit 922337203685477580 to my balance with no problem:

> 03:43 Banker: I think you must be one of the richest inhabitants in the world! Your account balance is 9.2233720368548e+17 gold.

so I'm taking out this limit

Reply:
wait, should I just remove the isValidMoney function and use isNumber instead?

Reply:
If you remove isValidMoney then I could transfer `-1000` to someone, effectively taking their money, right?

Reply:
🤔 right

Reply:
if it's stable you can display the number this way
```local n = 922337203685477580
print(string.format("%.0f",n))```

https://www.lua.org/cgi-bin/demo